src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -198,7 +198,10 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
     const connectedIntegration = getConnectedIntegration(policy, accountingIntegrations) ?? connectionSyncProgress?.connectionName;
 
     const policyID = policy?.id ?? '-1';
-    const successfulDate = getIntegrationLastSuccessfulDate(connectedIntegration ? policy?.connections?.[connectedIntegration] : undefined);
+    const successfulDate = getIntegrationLastSuccessfulDate(
```

Comment:
Can you add some comments here? It'll be nice to explain a little bit about how the `sussessfulDate` is determined.